cfgov/v1/models/molecules.py

```diff
@@ -12,7 +12,7 @@ def isRequired(field_name):
 
 
 class HalfWidthLinkBlob(blocks.StructBlock):
-    heading = blocks.CharBlock(max_length=100, required=True)
+    heading = blocks.CharBlock(max_length=100, required=False)
```

Comment:
If all the ` blocks.CharBlock` required flags are going to be defaulted to False, would it make sense to subclass `blocks.CharBlock?